src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -2324,18 +2324,21 @@ internal void TogglePreviewControlAllowance()
         /// </summary>
         private void StashNodeViewCustomizationMenuItems()
         {
-            foreach (var obj in grid.ContextMenu.Items)
+            if (MainContextMenu != null && MainContextMenu.Items.Count > 0 && NodeViewCustomizationMenuItems.Count < 1)
```

Comment:
Why don't you just say `NodeViewCustomizationMenuItems.Count == 0` instead of `< 1`? 

How is `NodeViewCustomizationMenuItems` different from `MainContextMenu.Items`?

Reply:
If `NodeViewCustomizationMenuItems` is expected to be less than 1, I don't believe this condition `` in line

Reply:
>Why don't you just say NodeViewCustomizationMenuItems.Count == 0 instead of < 1?

I just moved the existing condition around the method, into the method itself, will update that comparison.

>If NodeViewCustomizationMenuItems is expected to be less than 1, I don't believe this condition if (NodeContextMenuBuilder.NodeContextMenuDefaultItemNames.Contains(menuItem.Header.ToString())) in line 2334 can ever be true. I think it should be removed.

It is inside a loop, so it could have items at some point, and can be used to avoid some duplicates, (however that is unlikely as well). Let me know if you still think it should be removed and I will.